### ISSUE TITLE
ready to transfer to cbiomes organization?

### ISSUE THREAD

**Issue by gaelforget** [jpmattern/bayesian_cbiomes#12]
Now, leading to the virtual meet, seems like a good time to transfer this repo to https://github.com/CBIOMES

@jpmattern do you want to do that?

It's pretty easy, takes a few clicks, although you just want to be careful -- it's all explained in https://help.github.com/en/github/administering-a-repository/transferring-a-repository in case you have never done this

**Comment by gaelforget** [comment]
Could also be a good time to rename the repo too if we'd like. Something like `bayesian-workshop` maybe?

**Comment by gregbritten** [comment]
I agree it would be good to move before the online annual meeting. Maybe for a name
 `bayesian-workshop2020` 
since we are planning for more :-)

**Comment by gaelforget** [comment]
> bayesian-workshop2020

Sounds good. Might suggest adding a dash (`bayesian-workshop-2020`) but not a huge deal

**Comment by jpmattern** [comment]
Just saw this now, sure, I'll give it a go!

**Comment by jrcasey** [comment]
Could also keep the existing repo where it is and host the clean distribution repo on cbiomes? We should probably use that for the publication anyway...
